Tasks/Common/utility-common/kubectl-object-model.ts

```diff
@@ -133,6 +133,37 @@ export class Kubectl {
         return command.execSync();
     }
 
+    public delete(files: string | string[], types: string, names: string[], labels: string, cascade: boolean, gracePeriod: string) {
```

Comment:
You could just set type to be `-f` in case of files I suppose? Like in [PATCH](https://github.com/Microsoft/azure-pipelines-tasks/blob/c8d891242208f4141d9ccd545181bf9841f9d9a7/Tasks/KubernetesManifestV0/src/actions/patch.ts#L15)